cmd/kubeadm/app/apis/kubeadm/v1beta3/types.go

```diff
@@ -53,7 +53,8 @@ type InitConfiguration struct {
 	LocalAPIEndpoint APIEndpoint `json:"localAPIEndpoint,omitempty"`
 
 	// CertificateKey sets the key with which certificates and keys are encrypted prior to being uploaded in
-	// a secret in the cluster during the uploadcerts init phase.
+	// a secret in the cluster during the uploadcerts init phase. 
```

Comment:
why is this line changed?

Reply:
Not sure about this one .

Reply:
they are the same, byte-by-byte.
strange.